src/main/kotlin/org/jetbrains/research/testspark/display/coverage/CoverageVisualisationTabFactory.kt

```diff
@@ -228,6 +231,55 @@ class CoverageVisualisationTabFactory(private val project: Project, private val
         )
     }
 
+    private fun getPanel(data: ArrayList<String>): JScrollPane {
+        // Implementation of abstract table model
+        val tableModel = object : AbstractTableModel() {
+            /**
+             * Returns the number of rows.
+             *
+             * @return row count
+             */
+            override fun getRowCount(): Int {
+                return 1
+            }
+
+            /**
+             * Returns the number of columns.
+             *
+             * @return column count
+             */
+            override fun getColumnCount(): Int {
+                return 4
+            }
+
+            /**
+             * Returns the value at index.
+             *
+             * @param rowIndex index of row
+             * @param columnIndex index of column
+             * @return value at row
+             */
+            override fun getValueAt(rowIndex: Int, columnIndex: Int): Any {
+                return data[rowIndex * 4 + columnIndex]
+            }
+        }
+
+        val table = JBTable(tableModel)
+
+        val mainPanel = ScrollPaneFactory.createScrollPane(table)
+
```

Comment:
Would you mind placing a comment above the `mainPanel` mentioning that the below panel is for rendering coverage stats? So that it can be easily perceived at a glance.